scripts/ci/libraries/_verify_image.sh

```diff
@@ -19,11 +19,11 @@
 function verify_image::verify_prod_image {
     DOCKER_IMAGE="${1}"
     export DOCKER_IMAGE
-    "${SCRIPTS_CI_DIR}/images/ci_run_docker_tests.sh" "${AIRFLOW_SOURCES}/docker_tests/prod_image.py"
+    "${SCRIPTS_CI_DIR}/images/ci_run_docker_tests.py" "${AIRFLOW_SOURCES}/docker_tests/prod_image.py"
```

Comment:
I feel it’s better to call `python` explicitly here instead of relying on the shebang.

Reply:
Why the feeling ?

Reply:
Because shebang requires an additional trip to the kernel, and the `env` context is sometimes finnicky to debug if somethings goes wrong. It’s not exactly wrong (and definitely useful), but if we can be explicit, we should.

Reply:
Makes sense. Also We should change python to python3 I think @mik-laj - both in shebang and here.

Reply:
Done. We use python3 to start the script and python3 in shebang.